.circleci/config.yml

```diff
@@ -25,10 +25,12 @@ jobs:
            name: Run the tests for Python 2.7
            command: |
              tox -e py27
+           no_output_timeout: 30m
```

Comment:
Why installing these takes so long ? Since there is already wheels available it should be fairly quick.

Reply:
It started to happen this morning. Maybe circle invalidated their cache for the xenial docker images? For some reason pip installing scipy takes forever.

Reply:
I suggest to do a verbose install to find out more details. Even without cache, it should be quick.

I suspect the wheels end up being build from source ...

Reply:
I think you are right. Any ideas why that would happen all of a sudden?

Reply:
Could you point me to the verbose log ?

Reply:
Funny thing is it started to use the wheel again. I just rebuild without cache and it just got significantly faster. https://circleci.com/gh/NeurodataWithoutBorders/pynwb/168. I suggest we keep the timout flags there though in case it tries to build wheel again since we don't know what caused that. Does that makes sense?

Reply:
> we keep the timout flags

Sure. In that case, add a comment indicating why the timeout is there.

Ideally, timeout should be added to all CI.

>

Reply:
May be a comment like this:

```
no_output_timeout: 30m  # Install is expected to be quick. Timeout early in case there are some network issues.
```

Reply:
Can we do it on other CIs if this occurs? This was happening specifically because circle thinks if the task waits more than 10 minutes without stdouting anything the task is a zombie. While installation tox does not print out anything by default so it was waiting more than 10 minutes. It did not cause problems with others.

Reply:
> Can we do it on other CIs if this occurs?

Sounds good

> While installation tox does not print out anything by default

That is the type of info to specify in the comment.